web/src/lib/utils/app.ts

```diff
@@ -0,0 +1,18 @@
+// should be the same values as the one in perferences.store.ts
+interface ThemeSetting {
+  value: Theme;
+  system: boolean;
+}
+
+// should be the same values as the ones in constants.ts
+enum Theme {
```

Comment:
Does it not work to export these?

Reply:
Nope:
```bash
 9:07:52 PM [vite] Error when evaluating SSR module /src/lib/utils/app.ts:
 |- ReferenceError: localStorage is not defined
     at /usr/src/app/src/lib/utils/app.ts:14:21
```

Reply:
Can we tell svelte that this isn't a SSR module?